server/src/main/scala/actors/DrtStaticParameters.scala

```diff
@@ -0,0 +1,13 @@
+package actors
+
+import drt.shared.{MilliTimes, SDateLike}
+
+object DrtStaticParameters {
+  val expireAfterMillis: Int = 2 * MilliTimes.oneDayMillis
+
+  val liveDaysAhead: Int = 2
+
+  def time48HoursAgo(now: () => SDateLike): () => SDateLike = () => now().addDays(-2)
+
+  def timeBeforeThisMonth(now: () => SDateLike): () => SDateLike = () => now().startOfTheMonth()
```

Comment:
this function seems weirdly named - is it not returning the start of the current month?

Reply:
I've no idea! I've not touched any of this code other than to move into its own file